lib/emerald/parser.rb

```diff
@@ -57,7 +57,7 @@ def parse_number(source)
     end
 
     def parse_string(source)
-      pattern = /(["])(\\?.)*?\1/
+      pattern = /^"([^"\\]|\\.)*"$/
```

Comment:
This only works for strings on a single line. Is that OK? Do we support multi-line strings?